scripts/code-examples.js

```diff
@@ -0,0 +1,133 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+const rimraf = require('rimraf');
+
+const REPO_ROOT_DIR = __dirname.split('scripts')[0];
+const NEXT_EXAMPLES_DIR = path.join(REPO_ROOT_DIR, 'examples', 'next');
+const TMP_DIR_NAME = 'tmp';
+const TMP_DIR = path.join('examples', TMP_DIR_NAME);
+
+const HOT_WRAPPERS = ['@handsontable/react', '@handsontable/angular', '@handsontable/vue'];
+
+const [/* node bin */, /* path to this script */, shellCommand, hotVersion] = process.argv;
+
+// Function search recursively in the provided `dirPath` for the package.json file
+// and gets the project directory path.
+const getExamplesFolders = (dirPath, exampleFolders) => {
+  const files = fs.readdirSync(dirPath);
+
+  exampleFolders = exampleFolders || [];
+
+  files.forEach((file) => {
+    if (file !== 'node_modules' && fs.statSync(path.join(dirPath, file)).isDirectory()) {
+      exampleFolders = getExamplesFolders(path.join(dirPath, file), exampleFolders);
+      return;
+    }
+    if (file === 'package.json') {
+      exampleFolders.push(dirPath);
+    }
+  });
+
+  return exampleFolders;
+};
+
+const getHotWrapperName = (packageJson) => {
+  const { dependencies } = packageJson;
+
+  return HOT_WRAPPERS.find(wrapper => Object.keys(dependencies).find(dependency => dependency === wrapper));
+};
+
+const updatePackageJsonWithVersion = (projectDir, version) => {
+  const packageJsonPath = path.join(projectDir, 'package.json');
+  const packageJson = fs.readJsonSync(packageJsonPath);
+  const wrapper = getHotWrapperName(packageJson);
+
+  packageJson.version = version;
+  packageJson.dependencies.handsontable = version;
+  if (wrapper) {
+    // TODO: uncomment it when wrappers will be using the same versioning as Handsontable
+    // pJsonFile.dependencies[wrapper] = hotVersion;
+  }
+  fs.writeJsonSync(packageJsonPath, packageJson, { spaces: 2 });
+};
+
+const runNpmCommandInExample = (exampleDir, command) => {
+  // eslint-disable-next-line
+  console.log(chalk.yellow(`"${command}" STARTED IN DIRECTORY "${exampleDir}"`));
+  execSync(command, {
```

Comment:
In the monorepo scripts, I used `execa` as a process-runner - maybe we could used it here as well? You can even use the `spawnProcess` helper from `utils/processes.mjs`.

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/f91ded4fdc314994db132a6314cd0b5d0478045d

I used the `execa.sync` but not the `spawnProcess` helper because I wanted it to be synchronous.